source/frontend/commands/_commandDictionary.js

```diff
@@ -52,3 +52,17 @@ exports.setLogic = function (logicBlob) {
 		commandDictionary[commandKey].setLogic?.(logicBlob);
 	}
 }
+
+exports.updateCooldownMap = function(map) {
+	for (const commandKey in commandDictionary) {
+		map[commandKey] = commandDictionary[commandKey].cooldown;
+	}
+}
+
+exports.updatePremiumList = function(list) {
```

Comment:
This is good encapsulation, we might be able to refactor away from `getCommand()` exposing entire command objects soon.

This will also be a good place for us to have SKU and other dAPI premium logic in the future: https://discord.js.org/docs/packages/discord.js/14.19.3/CommandInteraction:Class#sendPremiumRequired